snakeviz/ipymagic.py

```diff
@@ -1,10 +1,19 @@
 import subprocess
 import tempfile
 import time
+from IPython.display import display, HTML
+import uuid
+import urllib
 
 __all__ = ['load_ipython_extension']
 
 
+JUPYTER_HTML_TEMPLATE = """
+<iframe id='{uuid}' frameborder=0 seamless width='100%%' height='1000'></iframe>
```

Comment:
Maybe put a prefix on this `id`, in older browsers this can error if the `id` starts with a number.